content/pt/docs/concepts/containers/runtime-class.md

```diff
@@ -0,0 +1,179 @@
+simples. Especifique um `runtimeClassName` na especificação do Pod. Por exemplo:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mypod
+spec:
+  runtimeClassName: myclass
+  # ...
+```
+
+Isso irá instruir o kubelet a usar o RuntimeClass nomeado acima (myclass) para esse Pod. Se
+o nome do RuntimeClass não exustir, ou o CRI não puder executar a solicitação, o Pod irá entrar na [fase
+final](/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase) `Failed`. Procure por um 
+[evento](/docs/tasks/debug-application-cluster/debug-application-introspection/) correspondente 
+para uma mensagem de erro.
+
+Se nenhum `runtimeClassName` for especificado, o RuntimeHandler padrão será utilizado, que é equivalente 
+ao comportamento quando a funcionalidade de RuntimeClass está desativada.
+
+### Configuração do CRI
+
+Para maiores detalhes de configuração dos agentes de execução CRI, veja [instalação do CRI](/docs/setup/production-environment/container-runtimes/).
+
+#### dockershim
+
+O CRI dockershim embutido no Kubernetes não suporta outros agentes de execução
+
+#### {{< glossary_tooltip term_id="containerd" >}}
+
+Agentes de execução são configurados através da configuração do containerd em 
+`/etc/containerd/config.toml`. Agentes válidos são configurados sob a seção de `runtimes`:
+
+```
+[plugins.cri.containerd.runtimes.${HANDLER_NAME}]
+```
+
+Veja a documentação de configuração do containerd para maiores detalhes:
+https://github.com/containerd/cri/blob/master/docs/config.md
+
+#### {{< glossary_tooltip term_id="cri-o" >}}
+
+Agentes de execução são configurados através da configuração do CRI-O em `/etc/crio/crio.conf`. 
+Agentes válidos são configurados na seção [crio.runtime
+table](https://github.com/cri-o/cri-o/blob/master/docs/crio.conf.5.md#crioruntime-table):
+
+```
+[crio.runtime.runtimes.${HANDLER_NAME}]
+  runtime_path = "${PATH_TO_BINARY}"
+```
+
+Veja a [documentação de configuração](https://raw.githubusercontent.com/cri-o/cri-o/9f11d1d/docs/crio.conf.5.md) do CRI-O para maiores detalhes.
+
+## Associação
+
+{{< feature-state for_k8s_version="v1.16" state="beta" >}}
+
+Ao especificar o campo `scheduling` para um RuntimeClass, você pode colocar limites e
+garantir que os Pods executando dentro de uma RuntimeClass sejam associados a nós que 
+suportem eles. Se o `scheduling` não estiver configurado, assume-se que esse RuntimeClass
+é suportado por todos os nós.
+
+Para garantir que os Pods sejam executados em um nó que suporte um RuntimeClass específico, 
+aquele conjunto de nós deve possuir uma marca/label padrão que é selecionado pelo campo 
+`runtimeclass.scheduling.nodeSelector`. O nodeSelector do RuntimeClass é combinado com o 
+nodeSelector do Pod em tempo de admissão, obtendo a intersecção do conjunto de nós selecionado 
+por cada. Se existir um conflito, o pod será rejeitado.
+
+Se os nós suportados possuírem marcação de restrição para previnir outros Pods com uma 
```

Comment:
Just a little typo: previnir to prevenir.

Reply:
```suggestion
Se os nós suportados possuírem marcação de restrição para prevenir outros Pods com uma 
```